### PR TITLE
Training Data Set Size Fix - from 0.3 to 0.2 

### PR DESCRIPTION
**Pull request by narayan-khanna**
## Summary by Sourcery

Tests:
- Decreased the test set size from 30% to 20% of the total dataset.

### CODE REVIEW

**Review by ayadhyadatamining — COMMENTED** [review]
Looks Good
